### ISSUE TITLE
Create Properties for neural network parameters in main.py

### ISSUE THREAD

**Issue by ttsuchi** [ttsuchi/neural-network-demo#10]
As was done for `pca.py`, we should create Kivy Properties for each of the neural network parameters (currently in `train.py`) in `main.py`.  For instance, "Hidden Units Learning Rate" should become a `BoundedNumericProperty` (with minimum of 0 and maximum of 1, say), and its value should be bound to the "hidden units learning rate" text box.
